code/modules/atmospherics/machinery/airalarm.dm

```diff
@@ -198,12 +204,11 @@
 		panel_open = TRUE
 		pixel_x = (dir & 3)? 0 : (dir == 4 ? -24 : 24)
 		pixel_y = (dir & 3)? (dir == 1 ? -24 : 24) : 0
+		update_icon()
```

Comment:
You shouldn't need this. `update_icon()` is called in `power_change()` which is called during init. Just remove this from new.